storage/store/inmemory/unsynchronized/transactions.go

```diff
@@ -0,0 +1,55 @@
+package unsynchronized
+
+import (
+	"sync"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+type Transactions struct {
+	//TODO: we don't need a mutex here as we have a guarantee by design
+	// that we write data only once and it happens before the future reads.
+	// We decided to leave a mutex for some time during active development.
+	// It'll be removed later on.
+	lock  sync.RWMutex
+	store map[flow.Identifier]*flow.TransactionBody
+}
+
+var _ storage.Transactions = (*Transactions)(nil)
+
+func NewTransactions() *Transactions {
+	return &Transactions{
+		store: make(map[flow.Identifier]*flow.TransactionBody),
+	}
+}
+
+// ByID returns the transaction for the given fingerprint.
+// Returns storage.ErrNotFound if transaction is not found.
```

Comment:
can you update the formatting of this to be similar to what you used here:
https://github.com/onflow/flow-go/pull/7205/files#diff-ba1d18889c4fd12549e98ba4eef9e08458d1ea3177b93c4fe51cb06ef232f4a8R50-R51